survey-runner.tf

```diff
@@ -459,6 +451,14 @@ module "author-api" {
       {
         "name": "DB_CONNECTION_URI",
         "value": "postgres://${var.author_database_user}:${var.author_database_password}@${module.author-database.database_address}:${module.author-database.database_port}/${var.author_database_name}"
+      },
+      {
+        "name": "RUNNER_SESSION_URL",
+        "value": "https://${var.env}-new-surveys.${var.dns_zone_name}/session?token="
```

Comment:
You should use the module output rather than rebuild the address
`"value": "${module.survey-runner-on-ecs.service_address}/session?token="`